packages/server-wallet/src/wallet/__test__/lock-app.test.ts

```diff
@@ -8,13 +8,19 @@ import {Store} from '../store';
 import {seedAlicesSigningWallet} from '../../db/seeds/1_signing_wallet_seeds';
 import adminKnex from '../../db-admin/db-admin-connection';
 import {testKnex as knex} from '../../../jest/knex-setup-teardown';
-import {defaultConfig} from '../../config';
+import {defaultConfig, extractDBConfigFromServerWalletConfig} from '../../config';
 
 import {stateVars} from './fixtures/state-vars';
 
 jest.setTimeout(10_000);
 
-const store = new Store(defaultConfig.timingMetrics, defaultConfig.skipEvmValidation);
+const store = new Store(
+  extractDBConfigFromServerWalletConfig(defaultConfig),
+  defaultConfig.timingMetrics,
+  defaultConfig.skipEvmValidation
+);
+
+afterAll(async () => store.knex.destroy());
```

Comment:
Suggestion: the `store` could have a `.destroy()` method that destroys its `knex`? The pattern being, a `wallet` destroys its direct descendants (`store`) and a `store` destorys its direct descendants (`knex`).

Reply:
SGTM, will update.

Reply:
520a43d68